### ISSUE TITLE
Is the Important statement correct or clear around Government clouds

### ISSUE THREAD

**Issue by TiJohn0108** [OfficeDev/office-js-docs-pr#4215]
In the Important statement it notes the following in part:
Customers in sovereign or **government clouds don't have access to Integrated Apps**. They **will use Centralized Deployment instead**. 

But if you look at the "Before you begin" in the Centralized Deployment (CD) article, Government licenses such as G3 and G5 aren't listed which would seem to imply that add-in deployment isn't available to Government in CD as well. It seems like some level of clarity can be added around what methods are available to Government tenants for add-in deployments.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 2677af5b-e9b8-f962-25ce-8aa285977c90
* Version Independent ID: c31c7ed9-0a0a-7a65-0d1b-70d6a8b33994
* Content: [Deploy and publish Office Add-ins - Office Add-ins](https://learn.microsoft.com/en-us/office/dev/add-ins/publish/publish)
* Content Source: [docs/publish/publish.md](https://github.com/OfficeDev/office-js-docs-pr/blob/main/docs/publish/publish.md)
* Service: **microsoft-365**
* Sub-service: **add-ins**
* GitHub Login: @o365devx
* Microsoft Alias: **o365devx**

**Comment by AlexJerabek** [comment]
Hi @TiJohn0108,

Thanks for asking this question. I'm going to investigate internally and get some clarity.

**Comment by AlexJerabek** [comment]
Hi @TiJohn0108,

G3/G5 licenses do support centralized deployment. The omission from that list in unintentional. 

I have a pending PR to correct the documentation and will close this issue when that gets checked in. Please let me know if you have any other questions.

**Comment by TiJohn0108** [comment]
Hello,

I guess that would be great but I'm seeing items elsewhere that seems to imply no such as:
If you still want to continue using Centralized Deployment for admin deployment of add-ins, navigate to Settings, then select Integrated Apps. On top of the Integrated Apps page, choose the Add-ins link. If Integrated Apps is not available for you, you should be able to navigate to Centralized Deployment from Settings, and then select Add-ins.
https://learn.microsoft.com/en-us/microsoft-365/admin/manage/manage-deployment-of-add-ins?view=o365-worldwide#deploy-an-office-add-in-using-the-admin-center

Yet Integrated Apps isn't available to GCC/Government Get started with Integrated apps - Microsoft 365 admin | Microsoft Learn<https://learn.microsoft.com/en-us/microsoft-365/admin/manage/test-and-deploy-microsoft-365-apps?view=o365-worldwide>

In short, clarity on what is in fact available to Government.

Thanks,
Tim

**Comment by TiJohn0108** [comment]
Tim Johnson would like to recall the message, "[OfficeDev/office-js-docs-pr] Is the Important statement correct or clear around Government clouds (Issue #4215)".

**Comment by AlexJerabek** [comment]
> Tim Johnson would like to recall the message, "[OfficeDev/office-js-docs-pr] Is the Important statement correct or clear around Government clouds (Issue #4215)".

I'm happy to delete that comment on your behalf, if that's the intention.

As for what's available on GCC, the article [Guidance for deploying Office Add-ins on government clouds](https://learn.microsoft.com/en-us/office/dev/add-ins/publish/government-cloud-guidance) should provide more information. In this case, Integrated Apps is the preferred deployment option, but it isn't available in government clouds. Centralized Deployment is still around to fill that government cloud need, and for legacy purposes. 

I do agree that the article [](https://learn.microsoft.com/en-us/microsoft-365/admin/manage/manage-deployment-of-add-ins?view=o365-worldwide) is confusing - the context of Integrated Apps and Centralized Deployment seems to switch. I'll work with the feature crew to improve that. I'm happy to field any questions here in the meantime.

**Comment by TiJohn0108** [comment]
Hello,

Sorry for the delay on getting back to this.

I’m not saying you are not correct around availability to GCC/Government clouds, but at least in my test GCC Tenant, just the general experience is different for Government Tenants as I see Add-ins under Settings in the left Nav Pane, and not Integrated Apps as I see in a non-Government test tenant.

At least how I’m seeing this, “Centralized Deployment” is “kind of” available to GCC, though the Store isn’t per “The ‎Microsoft 365‎ Store isn't covered by the ‎Office 365‎ Trust Center, or US Government Community Cloud and ‎Office 365‎ contractual commitments.​” So it appears that they can add add-ins via “Upload Customer Apps” path if they have access to a related URL path for that add in if available in AppSource or if they are provided the add-in’s XML from add-in provider.

I guess at least for me, long story short, just in general Centralized Deployment and Integrated Apps can just be confusing in general, and then throwing in what is or isn’t available around Government into the mix get’s it even more confusing.

I’m not sure how easy it is to navigate, map it out, the path to follow for Commercial and Government. It seems like that Government guidance article covers areas for them to consider in general for their environment and what isn’t available, but doesn’t seem to provide guidance on what in fact is available to them.

Sorry I’m not providing much guidance myself on this, I just know that I’m finding some of our statements are not super clear and I believe Government customers are finding the same.

Tim

**Comment by AlexJerabek** [comment]
Thank you for the feedback @TiJohn0108. I've passed along your thoughts to the team maintaining the CD/IA documentation. I think a big part of the confusion is there's not a clear distinction between "Centralized Deployment" (the feature used in GCC in lieu of Integrated Apps) and "centralized deployment" (the general concept of deploying add-ins to an organization). Effectively, what's listed in these articles is capital Centralized Deployment and available for GCC:
- [Deploy add-ins in the Microsoft 365 admin center](https://learn.microsoft.com/en-us/microsoft-365/admin/manage/manage-deployment-of-add-ins?view=o365-worldwide)
- [Manage add-ins in the Microsoft 365 admin center](https://learn.microsoft.com/en-us/microsoft-365/admin/manage/manage-addins-in-the-admin-center?view=o365-worldwide)

**Comment by AlexJerabek** [comment]
I'm also assigning @AkhileshShah-MS to help provide more clarity.

**Comment by TiJohn0108** [comment]
Thanks, I appreciate anything that can be done to add some level of clarity on availability and expectations of Office add-ins for Government/GCC customers (and customers as a whole).

And agreed on the references that are used as well what is seen in the UI and how that can change dependent on the type of tenant is in the picture.

And unfortunately, some of the notes that are added to the articles that should be a call out important items are not necessarily helping.
For example under https://learn.microsoft.com/en-us/microsoft-365/admin/manage/manage-deployment-of-add-ins?view=o365-worldwide#deploy-an-office-add-in-using-the-admin-center
Note
You can also deploy add-ins in the admin center through Integrated Apps<https://learn.microsoft.com/en-us/microsoft-365/admin/manage/test-and-deploy-microsoft-365-apps?view=o365-worldwide>. Integrated Apps is the recommended place for admins to deploy add-ins, and is available to Global Administrator, Azure Application Administrator and Exchange Administrator.
If you still want to continue using Centralized Deployment for admin deployment of add-ins, navigate to Settings, then select Integrated Apps. On top of the Integrated Apps page, choose the Add-ins link. If Integrated Apps is not available for you, you should be able to navigate to Centralized Deployment from Settings, and then select Add-ins.

First this note makes it sound like they have an option in deploying add-ins either using Integrated Apps and Centralized Deployment. Is there an actual option or is it rather based on the type of tenant on what actually is available or not? And what I have highlighted, it's stated if they want to use Centralized Deployment instead, again, sounding like they can make the choice (and maybe they can, but not apparent to me) the steps take them still through Integrated Apps; so I can see a customer going, okay, isn't that using Integrated Apps then and not what is supposedly Centralized Deployment??  And even the last sentence, which I believe relates to what might be seen in Government tenants, as Integrated Apps isn't available to them, it's still calling out Centralized Deployment.

It seems like it would be beneficial to have brief descriptions and perhaps link out to what are these two "options" if in fact they are options. Integrated Apps appears to be apps such as Teams Apps and Office Add-ins can be deployed via Integrated Apps. Integrated Apps doesn't appear to be available to Government so it would be via Add-ins, which is where they can see references to Centralized deployment of Add-ins.

Also related are references to Store (where I can see confusion on availability to Government as well) and Appsource, what is what, how are they different, is there a benefit of one over the other.

It's as if there's a glossary needed, what's available for different tenants and the related different paths.

Thanks,
Tim

**Comment by AlexJerabek** [comment]
Since the issue is in the microsoft-365-docs repo, and the feedback has been passed along. I'm going to close the issue here. Happy to continue answering questions as needed.
